src/averageAges.js

```diff
@@ -14,12 +14,29 @@
  *
  * @return {number}
  */
+
+const isMale = person => person.sex === 'm';
```

Comment:
let's use thes for beter visualization 
```suggestion
const isMale = (person) => person.sex === 'm';
```